tests/Fixer/FunctionNotation/NativeFunctionInvocationFixerTest.php

```diff
@@ -69,6 +69,53 @@ public function providerInvalidConfigurationElement()
         );
     }
 
+    public function testConfigureResetsExclude()
```

Comment:
Sounds like a good idea, but how can we do that?

Should we have an abstract test case for configurable fixers?